isamples_sesar/sample.py

```diff
@@ -400,7 +398,7 @@ class Sample(SQLModel, table=True):
     launch_type: Optional["Launch_Type"] = Relationship()
     nav_type: Optional["Nav_Type"] = Relationship()
     additional_names: Optional[List["Sample_Additional_Name"]] = Relationship()
-    sample_type: Optional["Sample_Type"] = Relationship(
+    sample_type: "Sample_Type" = Relationship(
```

Comment:
I'm curious why this changed.

Reply:
Every sample in sesar is required to have a sample type and certain assumptions (about a sample type existing) made in the transformer functions were giving errors in mypy.